tests/components/deconz/test_binary_sensor.py

```diff
@@ -179,10 +155,8 @@ async def test_add_new_binary_sensor(hass):
     gateway.api.event_handler(state_added_event)
     await hass.async_block_till_done()
 
-    assert "binary_sensor.presence_sensor" in gateway.deconz_ids
-
-    presence_sensor = hass.states.get("binary_sensor.presence_sensor")
-    assert presence_sensor.state == "off"
+    assert len(hass.states.async_all()) == 1
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6271139)